src/setup/cortx_deploy

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+DEFAULT_LOG_LEVEL=0
```

Comment:
Lets keep default at level 2, i.e. info
1 - DEBUG
2 - INFO
3 - WARNING
4 - ERROR 
5 - FATAL

Reply:
done , Have kept it default as $INFO (2)